airbyte-integrations/connectors/destination-redshift/src/main/resources/spec.json

```diff
@@ -45,6 +45,34 @@
         "description": "Whether or not to normalize the data in the destination. See <a href=\"https://docs.airbyte.io/architecture/basic-normalization\">basic normalization</a> for more details.",
         "title": "Basic Normalization",
         "examples": [true, false]
+      },
+      "access_key_id": {
```

Comment:
was running into some strange json schema stuff so put that on pause now - will relook tomorrow